features/update/async_accepted/feature.go

```diff
@@ -24,7 +24,6 @@ const (
 
 	succeed       UpdateDisposition = 0
 	failWithError UpdateDisposition = 1
-	failWithPanic UpdateDisposition = 2
```

Comment:
This now hangs or fails the workflow. Either way it doesn't fail the update anymore so removed form this test.